digits/dataset/generic/views.py

```diff
@@ -10,6 +10,7 @@
 import caffe_pb2
 import flask
 import matplotlib as mpl
+import matplotlib.pyplot as plt
```

Comment:
Nice! I tried but it seems Lint doesn't know that we can do that (and complains `F401 'matplotlib.pyplot' imported but unused`) so I think I'll leave it as is.